inc/shortcodes/class-shortcode.php

```diff
@@ -56,43 +56,64 @@ public static function callback( $attrs, $content = '' ) {
 	}
 
 	/**
-	 * Parse iframes from a string, if there are any
+	 * Parse a string of content for a given tag name.
 	 *
 	 * @param string $content
+	 * @param string $tag_name
 	 * @return array|false
 	 */
-	protected static function parse_iframes( $content ) {
+	protected static function parse_closed_tags( $content, $tag_name ) {
```

Comment:
Can we make this `private` ? I don't think I want to expose it as a public interface.